### DISCUSSION TITLE
186. Turning Our Data into DataLoaders with Manually Created Transforms - data normalization for downstream task

### DISCUSSION THREAD

**Discussion by abhishekmann** [mrdbourke/pytorch-deep-learning#542] [Q&A]
Hi Dan and fellow learners,

From what I have seen in practice is that even for a downstream dataset (dataset on which we are going to fine-tune our model), that the same mean and standard deviation are used for normalization as were used in pre-training.  However, does it not make more sense to use mean and stddev of the downstream dataset if the problem for which we are fine-tuning for is weakly related to the pre-training done. 

For example :  fine-tuning a model trained on ImageNet for a dataset of say chest X-ray, isn't it more rational to use the mean and stddev of chest X-ray dataset instead of ImageNet mean and stddev.

Apologies if it is a stupid question as I am not able to find a concrete answer anywhere.
